contracts/governance/InterchainGovernance.sol

```diff
@@ -191,7 +191,7 @@ contract InterchainGovernance is AxelarExecutable, TimeLock, Caller, IInterchain
     }
 
     /**
-     * @notice Making contact able to receive native value
+     * @notice Allow contract to receive native token
```

Comment:
Maybe native coin/ether/gas token? Token usually means ECR20, so I am not sure about this.

Reply:
Sure, fixed